tom_tns/templatetags/tns_extras.py

```diff
@@ -90,19 +107,40 @@ def classify_with_tns(context):
 
     # Get the spectra details from the latest spectra reduced datum or one passed in
     reduced_datum = None
-    if 'datum' in context:
+    if 'datum' in context and context['datum'].data_type == 'spectroscopy':
         reduced_datum = context['datum']
     else:
         spectra = target.reduceddatum_set.filter(data_type='spectroscopy')
         if spectra.exists():
             reduced_datum = spectra.latest()
     if reduced_datum:
-        initial['observation_date'] = reduced_datum.timestamp
+        hermes_datum_converter = get_hermes_data_converter_class()(validate=False)
```

Comment:
This will mean that this version of `tom_tns` will not be compatible with older versions of `tom_toolkit` prior to your new changes to `tom_base`.   We should make that clear somewhere.

Reply:
Yes once we cut a new version of the tom_base with the changes, I will add that as a requirement to this version of tom_tns